app/src/main/java/com/door43/translationstudio/core/TargetTranslation.java

```diff
@@ -185,6 +189,27 @@ public String getTargetLanguageId() {
     }
 
     /**
+     * get format of translation
+     * @return
+     */
+    public TranslationFormat getFormat() {
+        return mTranslationFormat;
+    }
+
+    private TranslationFormat readTranslationFormat() {
+        String formatStr = manifest.getString("format");
+        TranslationFormat format = TranslationFormat.get(formatStr);
+        if(null == format) {
```

Comment:
@neutrinog - Unfortunately this is a real world issue when importing old projects from SD card.  It looks like I will have to change the migration code to look at the manifest directly and not use TargetTranslation to identify the format.